pkg/db/db.go

```diff
@@ -5467,14 +5468,19 @@ func (h *DBHandler) RunCustomMigrationReleaseEnvironments(ctx context.Context) e
 	span, ctx := tracer.StartSpanFromContext(ctx, "RunCustomMigrationReleaseEnvironments")
 	defer span.Finish()
 	return h.WithTransaction(ctx, false, func(ctx context.Context, transaction *sql.Tx) error {
```

Comment:
with transaction must be called more than once

Reply:
done